cfunits-python/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --record record.txt
```

Comment:
No problem.  But py3k, setuptools, and easy udunits support via configparse seems like good reasons to switch to `cf_units` :wink: